native/src/modules/endpoint/sagas/watchContentLanguageSwitch.js

```diff
@@ -75,10 +74,8 @@ export function* switchContentLanguage(
     }
     yield put(insert)
   } catch (e) {
-    if (fromError(e) === ErrorCodes.NetworkConnectionFailed) {
-      // TODO IGAPP-498 The alert should be replaced with a snackbar, hence the TFunction should also be removed.
-      Alert.alert(t('languageSwitchFailedTitle'), t('languageSwitchFailedMessage'))
```

Comment:
Should we remove this translation?

Reply:
In general yes, but does this even work if the translations for malte are reimported afterwards?